pkg/reporter/reporter.go

```diff
@@ -4,26 +4,31 @@ import (
 	"github.com/google/osv-scanner/pkg/models"
 )
 
+// Reporter provides printing operations for vulnerability results and for runtime information (depending on the verbosity
+// level given to the Reporter implementation).
+//
+// When printing non-error-related runtime information, it is entirely up to the Reporter implementation as to where
+// the text is actually printed (if at all); in most cases for "human format" reporters this will be stdout whereas
+// for "machine format" reporters this will be stderr.
 type Reporter interface {
-	// PrintErrorf prints errors in an appropriate manner to ensure that results
+	// Errorf prints errors in an appropriate manner to ensure that results
 	// are printed in a way that is semantically valid for the intended consumer,
 	// and tracking that an error has been printed.
 	//
 	// Where the error is actually printed (if at all) is entirely up to the actual
 	// reporter, though generally it will be to stderr.
-	PrintErrorf(msg string, a ...any)
-	// HasPrintedError returns true if there have been any calls to PrintErrorf.
+	Errorf(format string, a ...any)
+	// HasPrintedError returns true if there have been any calls to Errorf.
 	//
 	// This does not actually represent if the error was actually printed anywhere
 	// since what happens to the error message is up to the actual reporter.
 	HasPrintedError() bool
```

Comment:
I think we should change this to `HasErrored` to mirror the rename from `PrintErrorf`, and that should also help make it clearer this isn't actually about if an error was printed but just "did an error happen at all?"

Reply:
I also renamed the field associated to it (in all `Reporter` implementations that have this field) from `hasPrintedError` -> `hasErrored` to keep it consistent 👍